### PR TITLE
Ensure CID 335767902 exists

### PR DESCRIPTION
**Pull request by we-ai**
This PR is to fix issue [#497](https://github.com/episphere/connectApp/issues/497), and ensures  CID 335767902 exists after logging in with a token.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -243,25 +243,15 @@ const userProfile = () => {
...
             if(parameters && parameters.token){
-                const response = await validateToken(parameters.token);
-                if(response.code === 200) {
-                    let obj = {
-                        335767902: (new Date(parseInt(user.metadata.a))).toISOString()
-                    }
-
-                    await storeResponse(obj);
-                }
+                await validateToken(parameters.token);
```
Comment (anthonypetersen) [inline app.js:248]: what if the token can't be validated?

Comment (anthonypetersen) [reply app.js:248]: if we aren't reacting to the result of validateToken() at all, it seems pointless to call it in the first place

Comment (we-ai) [reply app.js:248]: The `validateToken` function does more than validating the token. It also links uid to participant document. And current flow allows participant proceed, disregarding token valid or not. The only exception is 335767902, and this is why it is missing in some participant data.

Comment (anthonypetersen) [reply app.js:248]: makes sense, thanks for clarifying!

**Pull request merged**